pkg/cmd/enclave_secrets.go

```diff
@@ -120,6 +120,14 @@ func init() {
 	enclaveSecretsDownloadCmd.Flags().String("format", models.JSON.String(), "output format. one of [json, env]")
 	enclaveSecretsDownloadCmd.Flags().String("passphrase", "", "passphrase to use for encrypting the secrets file. the default passphrase is computed using your current configuration.")
 	enclaveSecretsDownloadCmd.Flags().Bool("no-file", false, "print the response to stdout")
+	// fallback flags
+	enclaveSecretsDownloadCmd.Flags().String("fallback", "", "path to the fallback file. encrypted secrets are written to this file after each successful fetch. secrets will be read from this file if subsequent connections are unsuccessful.")
```

Comment:
The `enclave secrets download` command currently panics when called due to these flags not being defined. It appears this has been broken since [v3.15.0](https://github.com/DopplerHQ/cli/releases/tag/3.15.0), which is when we introduced fallback support to `secrets download`. Given that we haven't heard from any customers, I would surmise that our deprecation message was successful in getting people to switch to the new command.